src/components/CategoryList/CategoryList.tsx

```diff
@@ -19,9 +22,11 @@ export const CategoryList: React.FC<CategoryListProps> = ({
   primary,
   categories,
   selectCategory,
+  status: categoriesStatus
 }) => {
   return (
     <CategoryListContainer>
+      {categoriesStatus === 'loading' && <PendingIndicator alt={"Pending indicator"} size={ICON_SIZE.MEDIUM_ICON} />}
```

Comment:
`categoriesStatus === 'loading'`

should be replaced with

`categoriesStatus === FETCH_STATUS.LOADING` 

All instances of string `"loading"` should be replaced with FETCH_STATUS.LOADING